openupgradelib/openupgrade.py

```diff
@@ -700,6 +764,12 @@ def rename_fields(env, field_spec, no_deep=False):
                     'new_pattern': "$$'%s'$$" % new_field,
                 }, (model, ),
             )
+        # Perform rename field on a custom view used on dashboard
+        # TODO: Rename when the field is part of a submodel (ex. m2one.field)
+        if version_info[0] > 9:
+            _rename_field_on_dashboard(
```

Comment:
```suggestion
        if version_info[0] > 9:
            dashboard_view_data = env["ir.ui.view.custom"].search([])
            _rename_field_on_dashboard(
```
this way you avoid repeating the "if version_info[0] > 9" comparison